packages/prop-house-webapp/src/components/Modal/index.tsx

```diff
@@ -1,48 +1,75 @@
 import classes from './Modal.module.css';
-import ReactDOM from 'react-dom';
-import xIcon from '../../assets/icons/x-icon.png';
-import React from 'react';
+import React, { Dispatch, SetStateAction } from 'react';
+import ReactModal from 'react-modal';
+import Button, { ButtonColor } from '../Button';
+import Divider from '../Divider';
+import { useTranslation } from 'react-i18next';
 
-export const Backdrop: React.FC<{ onDismiss: () => void }> = (props) => {
-  return <div className={classes.backdrop} onClick={props.onDismiss} />;
-};
+const Modal: React.FC<{
+  title: string | JSX.Element | boolean;
+  subtitle: string | JSX.Element | boolean;
+  image?: { src: string; alt: string } | any;
+  body?: string | JSX.Element | any;
+  button?: any;
+  secondButton?: any;
+  onRequestClose?: () => void;
+  showModal: boolean;
```

Comment:
Can we discard this prop? What I'm seeing is that in every use of these components, the same var is the conditional being used to show the component anyways. So if the initial conditioal is false, the below modal components would never show so why use it within them?

eg.
```.ts
// TopLevelComponent.tsx
{showErrorModal && <ErrorModal showModal={showErrorModal} .. >}

// ErrorModal.tsx
<Modal showModal={showErrorModal} ... >

// Modal.tsx
<ReactModal isOpen={showModal} ... />
```

So if `TopLevelComponent.tsx` had `showErrorModal` as false, it would never render `<ErrorModal>` in the first place. 

If I'm right, then we can set `isOpen={true}` in `Modal` and allow top level component no manage the render or not + we remove that prop `showModal` from all the supersets of modal components we have.

Reply:
removed: https://github.com/Prop-House/prop-house-monorepo/pull/386/commits/0e9dfed07a9450a41f020ce7a8cd42cfdce8910a & https://github.com/Prop-House/prop-house-monorepo/pull/386/commits/365b624a3794c2175b07c8c826f9c60ecbecab4a